style.css

```diff
@@ -0,0 +1,5 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
- [ ] I suggest that you also implement the requested requirements as it was required to give the heading a color by using a class 

> A .css file that includes one class with styles for your header (make it in your favorite color).

this will help you get used to using classes for styling